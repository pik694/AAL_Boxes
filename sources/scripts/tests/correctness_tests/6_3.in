6
100 46 11
49 68 90
44 13 93
49 27 83
73 100 15
10 7 62

