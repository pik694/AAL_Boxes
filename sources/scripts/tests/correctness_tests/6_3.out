5 0 2 4 3 1 
627325->459980
